examples/android/app/build.gradle

```diff
@@ -4,8 +4,8 @@ android {
     compileSdkVersion 27
     defaultConfig {
         applicationId "org.sfmldev.android"
-        minSdkVersion 14
-        targetSdkVersion 19
+        minSdkVersion 19
+        targetSdkVersion 26
```

Comment:
What impact does it have updating these versions?

Reply:
they essentially define the minimum Android version the app supports together with the API/header version it's working with. In this case it's targeting for stuff as defined in API 26, but versions down to 19 are supported with necessary adjustments/workarounds for those.